pytorch_lightning/callbacks/progress/tqdm_progress.py

```diff
@@ -261,14 +261,14 @@ def on_validation_start(self, trainer, pl_module):
     def on_validation_batch_end(self, trainer, pl_module, outputs, batch, batch_idx, dataloader_idx):
         if self._should_update(self.val_batch_idx):
             _update_n(self.val_progress_bar, self.val_batch_idx)
-            if trainer.state.fn == pl.trainer.states.TrainerFn.FITTING:
+            if trainer.state.fn == "fit":
```

Comment:
Curious. Why change this?

Reply:
This was missed in the previous PR, the change is consistent with the reasoning here: https://github.com/PyTorchLightning/pytorch-lightning/pull/11213#discussion_r779514176